fastlane/lib/fastlane/actions/swiftlint.rb

```diff
@@ -49,14 +49,14 @@ def self.run(params)
 
       def self.optional_flags(params)
         command = ""
-        command << " --path #{params[:path].shellescape}" if params[:path]
         command << supported_option_switch(params, :strict, "0.9.2", true)
         command << " --config #{params[:config_file].shellescape}" if params[:config_file]
         command << " --reporter #{params[:reporter]}" if params[:reporter]
         command << supported_option_switch(params, :quiet, "0.9.0", true)
         command << supported_option_switch(params, :format, "0.11.0", true) if params[:mode] == :autocorrect
         command << supported_no_cache_option(params) if params[:no_cache]
         command << " --compiler-log-path #{params[:compiler_log_path].shellescape}" if params[:compiler_log_path]
+        command << " #{params[:path].shellescape}" if params[:path]
```

Comment:
I would consider handling `paths` too. 
Additionally we may want to update docs and maybe even deprecate `path` parameter.

Reply:
can you help in making this succeed? or guide me in getting this merged

Reply:
TBH I am not maintainer and last time I tried contributing to this repo (without success) was ~2 years ago 🙈 
Nevertheless you should find most of the answers to this question in related docs: https://github.com/fastlane/fastlane/blob/master/YourFirstPR.md